deployment/ansible/group_vars/all

```diff
@@ -3,7 +3,9 @@ app_username: "vagrant"
 
 packer_version: "0.7.5"
 
-nodejs_version: 4.6.0
+nodejs_version: 4.6.2
+
+virtualenv_version: 15.1.0
```

Comment:
Is this useful, or was it an attempt to fix the bug that was ultimately fixed by removing ipython?  Not that it hurts anything, but might as well not add overrides if they're not accomplishing anything.